java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -170,19 +166,215 @@ public int hashCode() {
     }
   }
 
-  private static final class IntegerStatisticsImpl extends ColumnStatisticsImpl
+  /**
+   * Column statistics for List and Map types.
+   */
+  private static final class CollectionColumnStatisticsImpl extends ColumnStatisticsImpl
+      implements CollectionColumnStatistics {
+
+    protected long minimum = Long.MAX_VALUE;
+    protected long maximum = Long.MIN_VALUE;
+    protected long sum = 0;
+    protected boolean hasMinimum = false;
+    protected boolean overflow = false;
+
+    CollectionColumnStatisticsImpl() {
+      super();
+    }
+
+    CollectionColumnStatisticsImpl(OrcProto.ColumnStatistics stats) {
+      super(stats);
+      OrcProto.IntegerStatistics intStat = stats.getIntStatistics();
```

Comment:
This should be the collectionStatistics.

Reply:
yes, this was an oversight, sorry :(